exec/csnex/Exec.cs

```diff
@@ -43,7 +48,9 @@ void PUSHB()
 
         void PUSHN()
         {
-            throw new NotImplementedException(MethodBase.GetCurrentMethod().Name);
+            ip++;
+            uint val = Bytecode.Get_VInt(bytecode.code, ref ip);
+            stack.Push(new Cell(Number.FromString(bytecode.strtable[(int)val])));
```

Comment:
All these `(int)` casts bother me. Would it perhaps be better to have `Get_VInt` return an `int` instead of a `uint`? I think that would overall require fewer casts, because `int` seems like more of a "natural" type in C# instead of `uint` (like for indexing arrays and such).

Reply:
Changed to `int`, and removed all the associated casts.